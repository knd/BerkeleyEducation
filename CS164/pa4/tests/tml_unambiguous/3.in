<tml>
	<head>
		<body>hello</body>
	</head>
</tml>
