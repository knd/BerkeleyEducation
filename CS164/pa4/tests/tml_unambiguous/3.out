[(0, 3, 'OpenTag -> \\< Id Attributes \\>'),
 (0, 19, 'Doc -> Tag'),
 (0, 19, 'Tag -> OpenTag Content CloseTag'),
 (1, 2, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*'),
 (2, 2, 'Attributes -> '),
 (3, 3, 'Content -> Text'),
 (3, 3, 'Text -> '),
 (3, 6, 'OpenTag -> \\< Id Attributes \\>'),
 (3, 16, 'Content -> Content Tags'),
 (3, 16, 'Tag -> OpenTag Content CloseTag'),
 (3, 16, 'Tags -> Tag'),
 (4, 5, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*'),
 (5, 5, 'Attributes -> '),
 (6, 6, 'Content -> Text'),
 (6, 6, 'Text -> '),
 (6, 9, 'OpenTag -> \\< Id Attributes \\>'),
 (6, 13, 'Content -> Content Tags'),
 (6, 13, 'Tag -> OpenTag Content CloseTag'),
 (6, 13, 'Tags -> Tag'),
 (7, 8, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*'),
 (8, 8, 'Attributes -> '),
 (9, 9, 'Content -> Text'),
 (9, 9, 'Text -> '),
 (9, 10, 'Content -> Text'),
 (9, 10, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*'),
 (9, 10, 'RawText -> Textlet'),
 (9, 10, 'Text -> RawText'),
 (9, 10, 'Textlet -> Id'),
 (10, 13, 'CloseTag -> \\<\\/ Id \\>'),
 (11, 12, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*'),
 (13, 16, 'CloseTag -> \\<\\/ Id \\>'),
 (14, 15, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*'),
 (16, 19, 'CloseTag -> \\<\\/ Id \\>'),
 (17, 18, 'Id -> [a-zA-Z][a-zA-Z0-9:_-]*')]
Yes
